core/cloudflow-akka/src/main/scala/cloudflow/akkastream/AkkaStreamletLogic.scala

```diff
@@ -89,6 +88,12 @@ abstract class AkkaStreamletLogic(implicit val context: AkkaStreamletContext) ex
    */
   def getExecutionContext() = executionContext
 
+  /**
+   * Helper method to make it easier to start typed cluster sharding
+   * with an classic actor system
+   */
+  def clusterSharding() = ClusterSharding(system.toTyped)
+
```

Comment:
@RayRoestenburg how does this look for the cluster sharding helper method?

Reply:
Looks good!